src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private static final double PI = 3.14159265359;
```

Comment:
There is no need to create a custom PI constant, you can use Math.PI for your purposes